hazelcast/src/main/java/com/hazelcast/jet/impl/util/IOUtil.java

```diff
@@ -34,8 +34,6 @@
 
 public final class IOUtil {
 
-    private static final int BUFFER_SIZE = 1 << 14;
```

Comment:
I like the change, and I'm sure it's fine. But someone is going to ask if this is slower / different...

Reply:
For production code - the PythonServiceContext.java - It is copying only small script files.
The other change is in test, so it does not affect performance